task5/src/main/java/com/clouway/adapter/http/LoginController.java

```diff
@@ -35,6 +35,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String email = request.getParameter("email");
     String password = request.getParameter("password");
+    sessionRepository.cleanExpired();
```

Comment:
This will be removed from here. I will leave it only in LoginFilter, so each time login page is requested, the inactive sessions will be deleted from the DB.